src/app/cli/src/commands/system_api_server_gql_query_command.rs

```diff
@@ -12,15 +12,15 @@ use dill::Catalog;
 use super::{CLIError, Command};
 
 pub struct APIServerGqlQueryCommand {
-    base_catalog: Catalog,
+    server_catalog: Catalog,
```

Comment:
This clarification is a nice idea!

Reply:
I am also thinking about typed catalog (via new type idiom)